lib/public/views/Home/Overview/index.js

```diff
@@ -26,24 +26,21 @@ const PAGE_USED_HEIGHT = 155;
  * @param {Object} model Pass the model to access the defined functions
  * @return {vnode} Return the view of the page with the filtering options
  */
-const HomeOverviewScreen = (model) => {
+const HomeOverviewScreen = ({ logs: logsModel, runs: { overviewModel: runsOverviewModel } }) => {
```

Comment:
Doing so, I would rather take two parameters: logsModel and runsOverviewModel
Doing so, you are independent from the Model structure